sway/commands/sandbox_socket.c

```diff
@@ -0,0 +1,119 @@
+#define _XOPEN_SOURCE 700 // for strdup
+#include <unistd.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <strings.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <wayland-server-core.h>
+#include "sway/commands.h"
+#include "sway/security.h"
+#include "sway/tree/container.h"
+#include "sway/tree/view.h"
+#include "list.h"
+#include "log.h"
+#include "util.h"
+
+struct sandbox_socket {
+	char* path;
+	struct wl_event_source *src;
+	int fd;
+	uint32_t label_id;
+};
+
+static list_t *sandbox_sockets;
+
+static int fd_accept(int srv_fd, uint32_t mask, void *data) {
+	struct sandbox_socket *sock = data;
+	struct sockaddr_un name;
+	socklen_t slen = sizeof(name);
+
+	int cli_fd = accept(srv_fd, (struct sockaddr *)&name, &slen);
```

Comment:
It doesn't look like you use `name` after `accept`. I think you can simply drop it.

```suggestion
	int cli_fd = accept(srv_fd, NULL, NULL);
```